backend/src/main.py

```
@@ -1,13 +1,67 @@
+    """Manage application lifespan."""
+    # Startup
+    logger = logging.getLogger(__name__)
+    logger.info("=" * 50)
+    logger.info("UI Element Detection API Starting Up")
+    logger.info("=" * 50)
+    logger.info(f"API Version: {app.version}")
+    logger.info(f"Database: Connected to PostgreSQL")
+    
+    # Initialize S3 storage
+    try:
+        from src.storage import storage
```

Ruff (0.12.2)

`src.storage.storage` imported but unused

Remove unused import: `src.storage.storage`

(F401)